components/Collection.vue

```diff
@@ -417,7 +440,7 @@ export default {
 					instance: map,
 					rectangle: rect
 				};
-				this.setMapSize("300px");
+				this.setMapSize(this.leafletOptions.height, this.leafletOptions.width);
 			} catch (e) {}
 		},
 		setMapSize(height, width = null) {
```

Comment:
```suggestion
		setMapSize(height, width) {
```

Reply:
+ change 

`this.$refs.mapContainer.style.width = width ? width : 'auto';`

to

`this.$refs.mapContainer.style.width = width;`

It's not really necessary to check defaults twice in the code.

Reply:
Yes, `setMapSize` is only called that one time, but nevertheless I left the default parameter value in there on purpose because I generally liked it.
I'm fine with dropping it, but _if_ we do that I want change line 448 to `this.$refs.mapContainer.style.width = width;` too so that the default value logic is entirely outside of that function.

Reply:
Ah, your second comment wasn't displayed initially. I agree with you then. Having the logic outside is nicer.

Reply:
Agreed, see my comment some minutes later :-) In theory the two map functions could also be merged, but maybe some granularity is better for understanding the code...